src/NzbDrone.Core/Notifications/Webhook/WebhookPayload.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookPayload
+    {
+        public string EventType { get; internal set; }
+        public WebhookSeries Series { get; internal set; }
+        public WebhookEpisodeFile EpisodeFile { get; internal set; }
+        public WebhookEpisode Episode { get; internal set; }
```

Comment:
so here is what I'm thinkings, we break what is currently `EpisodeFile` into two parts, the release info and file info,

we keep the release info separate as single item (since even a season pack would only have 1 release info), release info would be `sceneName`, `quality`, `releaseGroup` etc.

and we embed the file info into each episode, some will have duplicated info but if a consumer cares they can match items using the file path.